068-access-policies-in-jwt.md

```diff
@@ -78,6 +78,41 @@ and for a string:
 
 The field name is configurable with default value `acls`.
 
+### ACL Syntax
+
+A single ACL has the syntax `CLUSTER_NAME:RESOURCE_TYPE:RESOURCE_SPEC:PERMITTED_ACTIONS`.
+The separator `:` is chosen because it is a common separator in the Unix world.
+
+If the ACLs are passed as a string, they are separated by `,`.
+i.e., `CLUSTER_NAME:RESOURCE_TYPE:RESOURCE_SPEC:PERMITTED_ACTIONS,CLUSTER_NAME:RESOURCE_TYPE:RESOURCE_SPEC:PERMITTED_ACTIONS`
+
+- `RESOURCE_TYPE` can be `topic` or `group`, or the shortened versions `t` and `g`.
+- `PERMITTED_ACTIONS` is a list of a subset of `read`, `write`, `create`, `delete`, `alter`, `alter_configs`, `describe`, `describe_configs`, `cluster_action`, `all`, or the shortened versions `r`, `w`, `c`, `d`, `a`, `ac`, `de`, `dc`, `ca`. The short version for `all` is `*`.
```

Comment:
Looks like we miss some operations here.
https://kafka.apache.org/documentation/#operations_in_kafka

Reply:
Thanks for noticing, added the missing ones.